### DISCUSSION TITLE
M3UPT Link not working

### DISCUSSION THREAD

**Discussion by Adosano** [LITUATUI/M3UPT#77] [Q&A] [answered]
Hey guys, ive tried using your iptv to acess portuguese television for my wife and it was working for a week, today it just stopped working? ive tried on ssiptv and kodi too but i just get an error.

the website is also offline

**Comment by LITUATUI** [comment, marked as answer]
Hi.

Yesterday night I was making some changes to the server when my Internet connection stopped working, it only returned to normal just now. Everything is back to normal.

**Reply by Adosano** [reply]
Thank you very much for the work you put into this, we live in the uk and without this she wouldnt have any acesss to her country TV.

shes now happily watching CNN/SIC
